staging/src/k8s.io/legacy-cloud-providers/aws/aws_loadbalancer.go

```diff
@@ -212,7 +222,26 @@ func (c *Cloud) ensureLoadBalancerv2(namespacedName types.NamespacedName, loadBa
 				frontendPort := mapping.FrontendPort
 				frontendProtocol := mapping.FrontendProtocol
 				nodePort := mapping.TrafficPort
-
+				healthCheckModify := func(mapping nlbPortMapping, tgrp *elbv2.TargetGroup) bool {
```

Comment:
why we have lambda function like this instead of just define a healthCheckModified variable directly =.=